CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 ## Release Hightlights
 
 ## Important Notes
-- [#335] The session expiry for the OIDC provider is now taken from the Token Response (expires_in) rather than from the id_token (exp) 
+- [#335] The session expiry for the OIDC provider is now taken from the Token Response (expires_in) rather than from the id_token (exp)
+- [#405](https://github.com/pusher/oauth2_proxy/pull/405) The `/sign_in` page now honors the `rd` query parameter, fixing the redirect after a successful authentication
```

Comment:
This needs to be under the `Changes since v5.0.0` section.

Reply:
Thank you, this has been moved!